components/AccountOverview/index.tsx

```diff
@@ -284,7 +288,11 @@ const AccountOverview: React.FC<AccountOverviewProps & { refetch: () => Promise<
         />
       ),
       [`${GraphQLSchema.AccountType.PolyjuiceCreator}`]: (
-        <Polyjuice script={account.script as PolyjuiceCreator['script']} scriptHash={account.script_hash} />
+        <Polyjuice
+          script={account.script as PolyjuiceCreator['script']}
+          scriptHash={account.script_hash}
+          domain={domain}
```

Comment:
`PolyjuiceCreator` won't have a .bit domain because it doesn't have an eth address